charts/openmetadata/templates/serviceaccount.yaml

```diff
@@ -4,9 +4,25 @@ kind: ServiceAccount
 metadata:
   name: {{ include "OpenMetadata.serviceAccountName" . }}
   labels:
-    {{- include "OpenMetadata.labels" . | nindent 4 }}
-  {{- with .Values.serviceAccount.annotations }}
+    app.kubernetes.io/name: '{{ include "OpenMetadata.name" .}}'
+    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+    app.kubernetes.io/instance: "{{ .Release.Name }}"
+    app.kubernetes.io/component: "openmetadata"
+    {{- if .Values.labels }}
+    {{- range $key, $val := fromYaml .Values.labels }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end}}
+    {{- end}}
+
   annotations:
+    {{- with .Values.serviceAccount.annotations }}
```

Comment:
We should only be adding annotations from `.Values.serviceAccount.annotations` if they are available!

Reply:
annotations will be only added from .Values.serviceAccount.annotations if they are available